lib/google/apis/core/json_representation.rb

```diff
@@ -55,7 +55,7 @@ def getter_fn(name)
           def if_fn(name)
             ivar_name = "@#{name}".to_sym
             lambda do |opts|
-              if opts[:skip_undefined]
+              if opts[:options][:user_options] and opts[:options][:user_options][:skip_undefined]
```

Comment:
Due to a change in how `representable` represents the options map, `:options` is always present, though `:user_options` is not.